### ISSUE TITLE
hicNormalize Behavior of operations on cool file

### ISSUE THREAD

**Issue by rysterzhu** [deeptools/HiCExplorer#869]
When hicNormalize normalizes cool, it seems use weight in cool for balance first, which results in most of the bin values becoming 0 when using the '-sz 1.0 --normalize smallest' parameter.I think that normalizing cool should be done without balance, or provide an optional parameter.

**Comment by lldelisle** [comment]
Indeed, by default all tools in HiCExplorer will use weights if available. If you don't want to use them, please use `hicConvertFormat` with the option `--load_raw_values` to get a cool file without weights.
